access/grpc/convert/convert.go

```diff
@@ -333,9 +333,9 @@ func BlockHeaderToMessage(b flow.BlockHeader) (*entities.BlockHeader, error) {
 	}, nil
 }
 
-func MessageToBlockHeader(m *entities.BlockHeader) (flow.BlockHeader, error) {
+func MessageToBlockHeader(m *entities.BlockHeader) (*flow.BlockHeader, error) {
 	if m == nil {
-		return flow.BlockHeader{}, ErrEmptyMessage
+		return &flow.BlockHeader{}, ErrEmptyMessage
```

Comment:
I think this can also be nil
```suggestion
		return nil, ErrEmptyMessage
```